src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -357,37 +357,35 @@ export const evidenceSummaryView = ({ formData }) => {
   );
 };
 
-const FullNameViewField = ({ formData }) => {
-  const { first, middle, last, suffix } = formData;
-  return <strong>{first} {middle} {last} {suffix}</strong>;
-};
-
-const SsnViewField = ({ formData }) => {
-  const ssn = formData.slice(5);
-  const mask = <span>•••-••-</span>;
-  return <p>Social Security number: {mask}{ssn}</p>;
-};
-
-const VAFileNumberViewField = ({ formData }) => {
-  const vaFileNumber = formData.slice(5);
-  const mask = <span>•••-••-</span>;
-  return <p>VA file number: {mask}{vaFileNumber}</p>;
-};
-
-const DateOfBirthViewField = ({ formData }) => {
-  return <p>Date of birth: <DateWidget value={formData} options={{ monthYear: false }}/></p>;
+/**
+ * @param {ReactElement|ReactComponent|String} srIgnored -- Thing to be displayed visually,
+ *                                                           but ignored by screen readers
+ * @param {String} substitutionText -- Text for screen readers to say instead of srIgnored
+ */
+const srSubstitute = (srIgnored, substitutionText) => {
```

Comment:
Have we already checked in with @1Copenut for advice on this?

Reply:
Nope. I asked in Ad Hoc's accessibility channel and that's the advice I got, but it would be great to get feedback from the guru himself.

What're your thoughts, @1Copenut?

Reply:
I don't see any issue with this pattern. You have a visible string for sighted users that screen readers will ignore, and a screen reader only string that sighted users will never see. Feel free to send me a URL and I can validate it against assistive devices as needed.